web/public/robots.txt

```diff
@@ -1,2 +1,2 @@
 User-agent: *
-Disallow: /
+Allow: /
```

Comment:
FWIW -- this we already permitted in 
- https://github.com/dandi/dandi-archive/pull/2200

which was released 3 days ago.

Reply:
updated to main

Reply:
@yarikoptic, I'm aware but "redoing" that small change here was easier than rebasing on master or merging it into this branch--it will not cause any problems when this is merged to master.